app/models/user.rb

```diff
@@ -32,14 +40,4 @@ def full_name
     return username unless first_name.present?
     "#{first_name} #{last_name}"
   end
-
-  protected
-
-  def password_required?
-    !password.nil? || !password_confirmation.nil?
```

Comment:
He's just removing this from the user model since it's also at the Authenticable concern. What this does is override a device method which checks if a password presence validation is needed or not.